modules/message_dispatcher.lua

```diff
@@ -103,7 +103,8 @@ function MD.FileStream(filename, version, sessionId, service, encryption, bandwi
   local frameSize = (constants.FRAME_SIZE["P" .. res.version]
       - constants.PROTOCOL_HEADER_SIZE)
 
-  res.chunksize = (frameSize < bandwidth) and frameSize or (bandwidth - 1)
+  local chunksize = (frameSize < bandwidth) and frameSize or (bandwidth)
+  res.chunksize = math.floor(chunksize / 2 + 0.5)
```

Comment:
@dboltovskyi please add comments why this operations was done.

Reply:
@LuxoftAKutsan Done in b301857